lib/howl/ui/editor.moon

```diff
@@ -212,6 +212,8 @@ class Editor extends PropertyObject
       start, stop = @selection\range!
       @buffer\chunk start, stop - 1
 
+  @property current_mode: get: => @buffer\mode_at @cursor.pos
```

Comment:
I think this name is confusing because the main mode is also 'current'. Maybe `mode_at_cursor`?